grouper/model_soup.py

```diff
@@ -861,6 +860,10 @@ def apply_changes(self, request):
             else:
                 setattr(self, key, value)
 
+    def apply_changes(self, request):
+        changes = json.loads(request.changes)
```

Comment:
I added a TODO for that because it felt beyond the scope of this PR. This deserialization existed in the model already (I just renamed it), and from what I've seen it'd be a non-trivial amount of work to remove it altogether because it's relied upon by several of the Group methods. This is probably (IMO) best fixed when Group is moved out of model_soup and into it's own file/